packages/settings-library/src/settings_library/email.py

```diff
@@ -18,3 +19,29 @@ class SMTPSettings(BaseCustomSettings):
     SMTP_TLS_ENABLED: bool = Field(False, description="Enables Secure Mode")
     SMTP_USERNAME: Optional[str]
     SMTP_PASSWORD: Optional[SecretStr]
+
+    @root_validator
```

Comment:
FYI: you can also use ``@validation("SMTP_USERNAME", "SMTP_PASSWORD")`` and you receive ``(v, values)`` as arguments ..

Reply:
👍 ahh you also need the `v`, good to know. I was missing it.